edenia/eden-spend-explorer
# feat(webapp): check texts and json

### Check texts and JSON

### What does this PR do?

- Resolve #243
- Resolve #197

### Steps to test

1. run the project and open localhost:3000
2. see the information and translation in the whole system 

#### CheckList
